src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pInputLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "inputLatency"));
+    m_pInputLatencyCompensationHeadphonesDelay = new EngineDelay(group,
+        ConfigKey(group, "headMixDelay"));
```

Comment:
That was only possible in Mixxx 2.0 by excluding the talkover mix from the master mix. Take another look at [the code](https://github.com/mixxxdj/mixxx/blob/1.12/src/engine/enginemaster.cpp#L523). This PR does not change that. If the user wishes to hear the master mix in headphones but not hear the talkover mix, they could select the direct monitor (record/broadcast only) talkover mix mode. Or they could use the master only mode and listen to the booth output with headphones.